letsencrypt/tests/crypto_util_test.py

```diff
@@ -212,6 +212,17 @@ def test_parse_no_sans(self):
         self.assertEqual(
             [], self._call(test_util.load_vector('csr-nosans.pem')))
 
+class PyOpenSslLoaderTest(unittest.TestCase):
+    def test_pyopenssl_load(self):
+        from letsencrypt.crypto_util import _pyopenssl_load
+
+        method_mock = mock.MagicMock()
+        fake_data = "this is test data"
+        fake_types = ('fake loading type',)
```

Comment:
nit: "Be consistent with your choice of string quote character within a file." as per https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Strings#Strings.